vm/c/vm.cpp

```diff
@@ -243,6 +258,21 @@ CallResult run_transaction(CallContext ctx) {
     auto log_pb = result.add_logs();
     k_to_log(log, log_pb);
   }
+
+  if (withCoverage) {
+    static uint32_t tag4 = getTagForSymbolName("LblextractCoverage{}");
+    arr[0] = (block *)final_config->children[2];
+    extractedCoverages* extractedCoverage = (extractedCoverages *)evaluateFunctionSymbol(tag4, arr);
+    auto bytecodecoverages = k_to_coverage(&extractedCoverage->bytecodecoverages->data);
+    for (bytecodecoverage_cell *bytecodecoverage : bytecodecoverages) {
+      auto coveragemsg = result.add_coveragedata();
+      std::string codehash = of_z_width(32, bytecodecoverage->bytecodeHash->data);
+      std::string coveragebytes = std::string(bytecodecoverage->coverageData->data->data, len(bytecodecoverage->coverageData->data));
+      coveragemsg->set_codehash(codehash);
+      coveragemsg->set_coverage(coveragebytes);
+    }
+  }
```

Comment:
@dwightguth this one block I don't feel very comfortable reviewing, could you or @theo25 take a look and make sure it makes sense?

Reply:
This looks fine but it looks like you're leaving the bytecode field of the protobuf message blank? Is that deliberate?

Reply:
At the moment yes, it's deliberate, although I guess it wouldn't hurt to add it in. I included the bytecode field in the spec since it might be needed by the Firefly frontend, but we haven't got that far yet.